src/main/java/io/penguinstats/controller/ReportController.java

```diff
@@ -48,9 +47,6 @@
 	@Autowired
```

Comment:
spring doesn‘t recommend filed inject，you can use @Setter(onMethod =@__(@Autowired)) instead